files/netdata-apcupsd.conf

```diff
@@ -0,0 +1,47 @@
+# you can disable an alarm notification by setting the 'to' line to: silent
+
+ template: apcupsd_10min_ups_load
+       on: apcupsd.load
+    class: Utilization
+     type: Power Supply
+component: UPS
+       os: *
+    hosts: *
+   lookup: average -10m unaligned of percentage
+    units: %
+    every: 1m
+     warn: $this > (($status >= $WARNING)  ? (70) : (80))
+    delay: down 10m multiplier 1.5 max 1h
+     info: average UPS load over the last 10 minutes
+       to: sitemgr
+
+# Discussion in https://github.com/netdata/netdata/pull/3928:
+# Fire the alarm as soon as it's going on battery (99% charge) and clear only when full.
+ template: apcupsd_ups_charge
+       on: apcupsd.charge
+    class: Errors
+     type: Power Supply
+component: UPS
+       os: *
+    hosts: *
+   lookup: average -60s unaligned of charge
+    units: %
+    every: 60s
+     warn: $this < 100
+     crit: $this < 40
+    delay: down 10m multiplier 1.5 max 1h
+     info: average UPS charge over the last minute
+       to: sitemgr
+
+ template: apcupsd_last_collected_secs
+       on: apcupsd.load
+    class: Latency
+     type: Power Supply
+component: UPS device
+     calc: $now - $last_collected_t
+    every: 10s
+    units: seconds ago
+     warn: $this > (($status >= $WARNING)  ? ($update_every * 5) : ( 5 * $update_every * 5))
```

Comment:
Is this the one that's been spamming for some reason, can we back off a bit?

Reply:
Yes this is the one that was spamming.  I set the warning threshold to 5 times the original one, so we get the email now at 250 seconds, which I think is reasonable, but we can set it to higher if you want.

Reply:
OK!